.travis.yml

```diff
@@ -1,77 +1,71 @@
 language: php
-
 sudo: false
-
 cache:
-
   directories:
     - $HOME/.composer/cache
+    - $HOME/symfony-bridge/.phpunit
 
 env:
-
   global:
     - SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
     - PHPUNIT_FLAGS="-v"
     - PHPUNIT_ENABLED="true"
     - SYMFONY_PHPUNIT_VERSION="6.5"
-    - PHPCSFIXER_ENABLED="false"
     - COVERALLS_ENABLED="false"
 
 matrix:
-
   fast_finish: true
-
   include:
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+      env:
+        - COVERALLS_ENABLED="true"
+        - PHPUNIT_FLAGS="-v --coverage-text --coverage-clover var/build/clover.xml"
 
+    # Minimum supported dependencies with the latest and oldest supported PHP versions
     - php: 7.1
+      env:
+        - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
```

Comment:
do we need --prefer-stable? i think not, it seems to be in composer.json (plus as you say, we should remove the minimum-stability setting in composer.json and then the flag becomes entirely obsolete)

Reply:
you'll need to change composer.json too